helium-lib/src/solana_transaction_utils/priority_fee.rs

```diff
@@ -0,0 +1,220 @@
+use itertools::Itertools;
+
+use crate::{
+    anchor_lang::ToAccountMetas,
+    client::SolanaRpcClient,
+    error::Error,
+    keypair::Pubkey,
+    solana_client,
+    solana_sdk::{
+        instruction::{AccountMeta, Instruction},
+        message::Message,
+        signers::Signers,
+        transaction::Transaction,
+    },
+    utils::replace_or_insert_instruction,
+};
+
+pub const MAX_RECENT_PRIORITY_FEE_ACCOUNTS: usize = 128;
+pub const MIN_PRIORITY_FEE: u64 = 1;
+
+pub async fn get_estimate_with_min<C: AsRef<SolanaRpcClient>>(
+    client: &C,
+    accounts: &impl ToAccountMetas,
+    min_priority_fee: u64,
+) -> Result<u64, Error> {
+    let client_url = client.as_ref().url();
+    if client_url.contains("mainnet.helius") {
```

Comment:
would this break if running against helius testnet; would it be safe to `if client_url.contains("helius")`?